### PR TITLE
Java8+updateproj

### PR DESCRIPTION
**Pull request by michaelmassee**
Hi,
update to Java 8  and 2 new Eclipse Projects for building a Feature and Update.
that is the way i do so with my Eclipse Plugins in the Company. 

when merge ?? , then  please check if all the ids and names are correct.

### CODE REVIEW

File: core/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -3,9 +3,9 @@ Bundle-ManifestVersion: 2
 Bundle-Name: LibreOffice eclipse integration - core
 Bundle-SymbolicName: org.libreoffice.ide.eclipse.core;singleton:=true
 Bundle-Vendor: The Document Foundation
-Bundle-Version: 3.0.0
+Bundle-Version: 3.0.1
```
Comment (smehrbrodt) [inline core/META-INF/MANIFEST.MF:6]: Please no version change in this patch. Version will be updated when a release is done.

File: java.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.libreoffice.ide.eclipse.java"
+      label="Loeclipse.Java"
+      version="3.0.1"
+      provider-name="The Document Foundation">
+
+   <description url="http://www.example.com/description">
```
Comment (smehrbrodt) [inline java.feature/feature.xml:8]: where is this domain shown?

**Review by smehrbrodt — CHANGES_REQUESTED** [review]
Thanks a lot! Just please no version update in this patch.

**Comment by smehrbrodt** [comment]
Any update here?
If you need help let me know.

**Comment by michaelmassee** [comment]
just cleanup the junit test, before i do some more bugfixes

**Pull request closed without merging**